training/xgboost/structured/base/setup.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from setuptools import find_packages
+from setuptools import setup
+
+# While this is an xgboost sample, we will still require tensorflow and
+# scikit-learn to be installed, since the sample uses certain functionalities
+# available in those libraries:
+#    tensorflow: mainly to copy files seamlessly to GCS
+#    scikit-learn: the helpfer functions it provides, e.g. splitting datasets
+
+REQUIRED_PACKAGES = [
+    'tensorflow==1.13.1',
+    'scikit-learn==0.20.2',
+    'pandas>=0.20',
+    'xgboost==0.81',
+    'cloudml-hypertune',
```

Comment:
Does hyper tuning making sense for a base XGBoost model?

Reply:
Good observation. Gonzalo noticed as he was developing the TF sample, that the base sample was very similar to the other samples (gpu, tpu, hp tuning). He was thinking of writing the sample once and turning on and off different parts of the code using arguments passed to the code.

I have not started the hp tuning sample, but I think I will face the same thing soon, which is why I have not removed the hp tuning code from the base sample just yet. Once I get to the point of writing the hp tuning sample, I will need to come back to this sample and refactor the code a little bit.